build/package/windows/kbc.wxs

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<?ifndef ProductVersion?>
+<?error ProductVersion property not defined?>
+<?endif?>
+
+<!-- Define a unique UpgradeCode per platform -->
+<?if $(var.Platform) = "x64"?>
+<?define InstallerVersion = "200"?>
+<?define UpgradeCode = "25389d03-f8c7-415e-9755-ce0666951c1f"?>
```

Comment:
Jako úplně přesně tomu nerozumím, ale má to umožnit nějaký major updaty do budoucna (podle https://wixtoolset.org/documentation/manual/v3/howtos/updates/major_upgrade.html). Možná to nepotřebujem, ale kdo ví. 🙂  (Vygeneroval jsem je tady https://www.guidgen.com/)